config/config.go

```diff
@@ -25,6 +25,8 @@ type WatchdogConfig struct {
 	// to prevent transfer type of chunked encoding
 	// which some servers do not support.
 	BufferHTTPBody bool
+
+	ConcurrencyLimit int
```

Comment:
Please can you add a brief comment in a similar style to the BufferHTTPBody?

Reply:
I'm OK with this  name, another name could be `max_inflight`. This is something also used in NATS Streaming, another component making up the OpenFaaS stack.

From the PR description:

> as they exceed the maximum number of in-flight requests

Reply:
Changed the name.